presto-hive/src/main/java/com/facebook/presto/hive/ParquetRecordCursorProvider.java

```diff
@@ -49,7 +55,7 @@
             DateTimeZone hiveStorageTimeZone,
             TypeManager typeManager)
     {
-        if (!isDeserializerClass(schema, ParquetHiveSerDe.class)) {
+        if (!PARQUET_SERDE_CLASS_NAMES.contains(getDeserializerClassName(schema))) {
```

Comment:
Should this check be based on the `getSimpleName()`, or are those the only two packages that will contain one of these?